homeassistant/components/pi_hole/services.yaml

```diff
@@ -1,9 +1,14 @@
 disable:
+  name: Disable
   description: Disable configured Pi-hole(s) for an amount of time
+  target:
+    entity:
+      integration: pi_hole
+      domain: switch
   fields:
-    entity_id:
-      description: Target switch entity
-      example: switch.pi_hole
     duration:
+      name: Duration
       description: Time that the Pi-hole should be disabled for
       example: "00:00:15"
+      selector:
+        text:
```

Comment:
We could use `time:` but I believe it would be confusing to the user to see AM/PM for a timer.